### ISSUE TITLE
sudo on OS X requires a password

### ISSUE THREAD

**Issue by m-dierker** [mitmproxy/mitmproxy#270]
https://github.com/mitmproxy/mitmproxy/blob/master/libmproxy/platform/osx.py#L13

tl;dr: This command requires a password on OS X 10.9.2, so it fails if the user hasn't used sudo in the last 5 minutes or so.

Longer Version:
3:45 AM: My arpspoofing class project works with mitmproxy, I've hijacked my Chromebook, yay!
_unrelated code changes_
3:55 AM: I run it again, "Transparent mode failure: could not resolve original destination" :( Well, it must be my fault, `git stash`, nothing changes, :(
_begins tracing through libmproxy code_
4:00 AM: Hmm.... they're using sudo to access the packet firewall text to parse.
4:01 AM: _runs `sudo -n /sbin/pfctl -s state` in terminal_... `sudo: a password is required`
4:02 AM: Wait... how are they bypassing the password? `man sudo`, `/-n`, `"If a password is required for the command to run, sudo will display an error message and exit."`
4:03 AM: _cries_
4:10 AM: Starts writing sleepy bug report

In short, there needs to be another way of accessing the pfctl information needed without a password. It shouldn't be terribly hard to write a quickfix for this project, but a long term fix would be good! :D
